doc/v1/documentation_data_format.rst

```diff
@@ -64,7 +64,7 @@ and
 - (optional) A visualization file, which contains specifications how the data
   and/or simulations should be plotted by the visualization routines [TSV]
 
-.. figure:: gfx/petab_files.png
+.. figure:: ../gfx/petab_files.png
```

Comment:
Might make sense to put `gfx` inside `v1`/`v2` since they will differ between versions.

Reply:
I considered that, but then thought it might be more convenient to have all figures in here. No strong opinion.

Reply:
Fine for me either way, but then the files should at least be named like `v1_petab_files.png`